test/ERC20RootVault.test.ts

```diff
@@ -664,6 +612,112 @@ contract<ERC20RootVault, DeployOptions, CustomContext>(
             });
 
             describe("edge cases:", () => {
+                describe("when lpPriceD18 <= hwmsD18", () => {
+                    it("do not charge performance fees", async () => {
+                        await this.erc20RootVaultGovernance
+                            .connect(this.admin)
+                            .stageDelayedProtocolParams({
+                                managementFeeChargeDelay: 0,
+                                oracle: this.mellowOracle.address,
+                            } as DelayedProtocolParamsStruct);
+
+                        await sleep(this.governanceDelay);
+                        await this.erc20RootVaultGovernance
+                            .connect(this.admin)
+                            .commitDelayedProtocolParams();
+
+                        var signer = await addSigner(randomAddress());
+                        const amount = BigNumber.from("3000000000000000010001");
+                        await preprocessSigner(signer, amount);
+                        await this.subject
+                            .connect(signer)
+                            .deposit(
+                                [
+                                    BigNumber.from("32000000000000000"),
+                                    BigNumber.from("32000000000000000"),
+                                ],
+                                DEFAULT_MIN_LP_TOKEN,
+                                []
+                            );
+
+                        await this.subject
+                            .connect(signer)
+                            .deposit(
+                                [
+                                    BigNumber.from("32000000000000000"),
+                                    BigNumber.from("32000000000000000"),
+                                ],
+                                DEFAULT_MIN_LP_TOKEN,
+                                []
+                            );
+
+                        await this.subject
+                            .connect(signer)
+                            .deposit(
+                                [
+                                    BigNumber.from("30000000000000000"),
+                                    BigNumber.from("30000000000000000"),
+                                ],
+                                DEFAULT_MIN_LP_TOKEN,
+                                []
+                            );
+                    });
```

Comment:
Do you check that balanceOf strategyPerformanceTreasury remains const or zero after all deposits?